rclcpp_lifecycle/README.md

```diff
@@ -2,6 +2,15 @@
 
 Package containing a prototype for lifecycle implementation. Visit the [design document](https://design.ros2.org/articles/node_lifecycle.html) for more information about this package.
 
+- [Lifecycle states](include/rclcpp_lifecycle/state.hpp) Define the State class
```

Comment:
Move to a FEATURES.md and link to docs.ros2.org.

Reply:
rclcpp and rclcpp_action have the features in the doxygen. I will follow the same structure.